static/elements/chromedash-form-field.js

```diff
@@ -54,12 +54,15 @@ export class ChromedashFormField extends LitElement {
 
   render() {
     const fieldProps = ALL_FIELDS[this.name] || {};
+    const labelProp = fieldProps.label;
+    const label = labelProp ? `${labelProp}:` : '';
```

Comment:
I think these two lines could be simpler if done like the way you did helpText:
`const label = fieldProps.label || '';`

Reply:
We need to add the ':' but only if fieldProps.label is defined.   Django adds the ':', I believe, when it uses the label if provided. So we have to do the same here.

I can duplicate the fieldProps.label expression and drop the labelProp var.